environments/staging/terraform.yml

```diff
@@ -253,7 +253,7 @@ servers:
     volume_encrypted: yes
     volume_type: "gp3"
     block_device:
-      volume_size: 5
+      volume_size: 10
```

Comment:
Are you aware if log rotation is enabled on these machines?

Reply:
This block storage is solely for kafka data (/opt/data/) whereas kafka logs are in /var/log/kafka, so I don't think that was the issue here. I don't know the details of kafka's log rotation, but it looks like we have a default of 28 days (kafka_log_retention) so I would guess that the answer is still that log rotation is enabled on staging.